source/2019-08-16-the-ember-times-issue-111.md

```diff
@@ -32,13 +38,16 @@ responsive: true
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+<!--alex disable king-queen-->
+## [New Empress Zoey 📷](https://twitter.com/melaniersumner/status/1156652744092069888)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+<div class="blog-row">
+  <img class="float-right small transparent padded" alt="Empress Zoey" title="Empress Zoey" src="/images/tomsters/empress.png" />
+
+You might recall that we wrote about [@mansona](https://github.com/mansona)'s talk about [Empress](https://www.youtube.com/watch?v=WL66D0bR0RQ&feature=youtu.be&t=780), Ember's static site/JAMStack product, last week. Hopefully you'll be inspired to build your own empress-blog template even if you only know HTML and CSS. **But wait, there's more!** 
```

Comment:
```suggestion
You might recall that we wrote about [@mansona](https://github.com/mansona)'s talk about [Empress](https://www.youtube.com/watch?v=WL66D0bR0RQ&feature=youtu.be&t=780), Ember's static site/JAMStack product, [last week](https://the-emberjs-times.ongoodbits.com/2019/08/09/issue-110). Hopefully you'll be inspired to build your own empress-blog template even if you only know HTML and CSS. **But wait, there's more!** 
```

Reply:
Should we add this? ^

Reply:
couldn't commit suggestion after accepting first one, so did manually

thanks, great edits!